// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/parser/ecmascript5/ComputedPropertyNames/parserES5ComputedPropertyName9.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,9 @@
 
 
 //// [parserES5ComputedPropertyName9.d.ts]
 declare class C {
+    [e]: Type;
 }
 
 /// [Errors] ////
 
